regulations/management/commands/setup_cors.py

```diff
@@ -17,7 +17,7 @@ def handle(self, *args, **options):
             CORSConfiguration={
                 'CORSRules': [
                     {
-                        'AllowedMethods': ['PUT'],
+                        'AllowedMethods': ['GET', 'PUT'],
```

Comment:
Would it make sense to have a separate bucket for the two types of content? I'm a bit apprehensive about allowing GET access to partially-uploaded docs.